### PR TITLE
fixing title of the text box in edit collection page

### LINKED ISSUE #5395: Edit collection page - The title of the text box is "User comment" instead of "Add-on note"
STR:
1. Load AMO dev and go to a collection
2. Click on Edit this collection and try to make a comment for an extension/theme from the list
3. Notice that the name of the text box is "User comment"
4. Save the comment and then click on Save collection
5. Observe the name of the text box

Expected result:
In both edit and view collection - "Add-on note" is displayed - as mentioned[ here]( https://user-images.githubusercontent.com/6797635/33636526-6ef3ae78-d9d1-11e7-9a80-5ce5bf85717b.png
)

Actual result:
There's an inconsistency between the two pages - Edit collection displays "User comment" instead of "Add-on note".

Notes:
This issue is reproducible on AMO dev with FF60(Win10). 
add-on note2

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #5395 
Edit View Mode
edit mode

Collection View
view

### CODE REVIEW

File: src/amo/components/EditableCollectionAddon/index.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ export class EditableCollectionAddonBase extends React.Component<
           <div className="EditableCollectionAddon-notes">
             <h4 className="EditableCollectionAddon-notes-header">
               <Icon name="comments-blue" />
-              {i18n.gettext('User comment')}
+              {i18n.gettext('Leave a note')}
```
Comment (bobsilverberg) [inline src/amo/components/EditableCollectionAddon/index.js]: This should be "Leave a note" when in edit note mode for the note, i.e., when editing a note or creating a new note, but it should be "Collector's note" when in view mode for the note, i.e., when in edit mode for the collection, but not yet editing the individual note.

Comment (bobsilverberg) [reply src/amo/components/EditableCollectionAddon/index.js]: Yes, that's correct. 😄

**Comment by championshuttler** [comment]
@bobsilverberg I am confused little bit how to proceed for this , can you please  guide me little bit?

**Comment by championshuttler** [comment]
@bobsilverberg I am confused little bit how to proceed for this , can you please  guide me little bit?

**Comment by bobsilverberg** [comment]
@championshuttler Your initial PR was a good start. The only difference is that we need to replace the "User comment" title with "Leave a note" when the user is editing the note, like in the sceenshot below:

screenshot 2018-07-16 10 17 35

but we need to replace the "User comment" title with "Collector's note" when the user is not editing the note, like in the screenshot below:

screenshot 2018-07-16 10 18 49

Please let me know if you have any specific questions about this.

**Comment by championshuttler** [comment]
Hi @bobsilberberg , can I work on it next week , my laptop is in service
center due to some problems. Really sorry for the delay :/

**Comment by rebmullin** [comment]
hey @championshuttler, how are you making out with this one? Please let us know if you need any help :)

**Comment by bobsilverberg** [comment]
@championshuttler Are you still working on this one? Is there anything you need any help with?

**Comment by muffinresearch** [comment]
@championshuttler since it's a while since we've heard from you I'm going to close this for now, but please re-open this if you have additional updates.

**Pull request closed without merging**